src/localization/index.ts

```diff
@@ -1,15 +1,15 @@
 export const langNames = {
-  en: '🇺🇸',
   de: '🇩🇪',
+  en: '🇺🇸',
   es: '🇪🇸',
   fr: '🇫🇷',
-  tr: '🇹🇷',
-  ru: '🇷🇺',
-  'zh-cn': '🇨🇳',
-  uk: '🇺🇦',
   ko: '🇰🇷',
   pl: '🇵🇱',
-  'pt-br': '🇵🇹',
+  'pt-br': '🇧🇷',
```

Comment:
@aykutkardas just to point out, I'm changing to the correct flag, possibly you did change It by mistake on commit ea07668. The one you used Is from Portugal :)

Reply:
You're right, I'm glad you corrected me. But I'd prefer if you didn't change the order of the flags. ✨